src/components/Widgets/RescueTime.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+    <div class="rescuetime-wrapper">
+    <div class="title-row">
+        <p class="title-rank">Rank</p>
+        <p class="title-name">Category</p>
+        <p class="title-time">Time spent</p>
+    </div>
+      <div
+        v-for="(category, indx) in categories"
+        :key="indx"
+        class="category-row"
+      >
+        <p class="category-rank">{{ category.rank }}</p>
+        <p class="category-name">{{ indx }}</p>
+        <p class="category-time">{{ category.minutes }} min</p>
+      </div>
+    </div>
+    </template>
+<script>
+
+// import axios from 'axios';
+import WidgetMixin from '@/mixins/WidgetMixin';
+import { widgetApiEndpoints } from '@/utils/defaults';
+
+export default {
+  mixins: [WidgetMixin],
+  data() {
+    return {
+      categories: [],
+    };
+  },
+  mounted() {
+    this.checkOptions();
+  },
+  computed: {
+    endpoint() {
+      const todaystring = this.getDate();
+      return `${widgetApiEndpoints.rescueTime}?key=${this.options.apiKey}&restrict_begin=${todaystring}&restrict_end=${todaystring}&restrict_kind=overview&format=json`;
+    },
+  },
+  methods: {
+    fetchData() {
+      this.makeRequest(this.endpoint).then(this.processData);
+    },
+    processData(data) {
+      const formateddata = this.calculateTimeCategories(data);
+      this.categories = formateddata;
+    },
+    checkOptions() {
+      const ops = this.options;
+      if (!ops.apiKey) this.error('Missing API key for RescueTime');
+    },
+    getDate() {
+      const today = new Date();
+      let day = today.getDate();
+      let month = today.getMonth() + 1;
+      const year = today.getFullYear();
+      if (day < 10) {
+        day = `0${day}`;
+      }
+      if (month < 10) {
+        month = `0${month}`;
+      }
+      return `${day}-${month}-${year}`;
+    },
+    calculateTimeCategories(timeArray) {
+      const results = {};
+      // console.log(timeArray.rows[0]);
```

Comment:
Forgot to remove this?
And the `// import axios from 'axios';` on L21 ;)

Reply:
Yep... looked over this about 10 times, somehow missed it. Thanks for removing it :)